RussianFilter/sections/specific.txt

```diff
@@ -32,6 +32,8 @@ nikolaevka24.com###med-1[style="margin-bottom: 20px;cursor:pointer"]
 49000.com.ua##.custom-html-widget > a[rel="nofollow"] > img
 zak-kor.net##.topblock > center > a[target="blank"] > img
 zora-irpin.info###execphp-8
+||cdn.hunterdelivery.com/scripts/draw_script.js$domain=versii.if.ua
```

Comment:
It seems this is ad domain. Better `||hunterdelivery.com^$third-party`

Reply:
@Alex-302,
moved to `RussianFilter/adservers.txt`